ui/src/album/GridMenu.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import IconButton from '@material-ui/core/IconButton'
+import Menu from '@material-ui/core/Menu'
+import MenuItem from '@material-ui/core/MenuItem'
+import MoreVertIcon from '@material-ui/icons/MoreVert'
+import { useDispatch } from 'react-redux'
+import { playAlbum, shuffleAlbum } from '../audioplayer'
+import { useGetList } from 'react-admin'
+
+const GridMenu = (props) => {
+  const [anchorEl, setAnchorEl] = React.useState(null)
+  const dispatch = useDispatch()
+  const { ids, data, loading, error } = useGetList(
```

Comment:
I need to think about this. Let me do some tests.

Reply:
Actually it could be called when the `handleItemClick` is called. But I don't think we will be able to use useGetList in this case. We may need to use [useDataProvider](https://marmelab.com/react-admin/Actions.html#usedataprovider-hook) to get a dataProvider, and then call it inside the event handler

Reply:
This works well now. The only part I did not implement is on what to do with error handling. Since we are not rendering anything, I think we can leave it as is unless we want to log it somewhere.